src/status_im/utils/ethereum/core.cljs

```diff
@@ -0,0 +1,69 @@
+(ns status-im.utils.ethereum.core
+  (:require [clojure.string :as string]
+            [status-im.js-dependencies :as dependencies]))
+
+;; IDs standardized in https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids
+
+(def chain-ids
+  {:mainnet 1
+   :ropsten 3
+   :rinkeby 4})
+
+(defn testnet? [id]
+  (contains? #{(:ropsten chain-ids) (:rinkeby chain-ids)} id))
+
+(defn network-name [id]
```

Comment:
it would be simpler to make another map of id to network-name for this and do (get network-names id "Unknown")

Reply:
Makes sense